src/Request/RequestHandler.php

```diff
@@ -139,42 +108,13 @@ public function handleRequest(string $method, string $uri = '', array $options =
     }
 
     /**
-     * Get token for Clarifai API requests
+     * Get API Key for Clarifai API Requests
      *
      * @return string
      */
-    private function getAuthToken()
+    private function getApiKey()
     {
-        // Generate a new token if current is expired or empty
-        if (!$this->token || new \DateTime() > $this->tokenExpireTime) {
-            $this->requestToken();
-        }
-
-        return $this->tokenType . ' ' . $this->token;
-    }
-
-    /**
-     * Make request to Clarifai API for the new token
-     */
-    private function requestToken()
-    {
-        $tokenResponse = $this->handleRequest(
-            'POST',
-            $this->url . '/v1/token', // endpoint is available only in v1
-            [
-                'form_params' => [
-                    'grant_type' => 'client_credentials',
-                    'client_id' => $this->clientId,
-                    'client_secret' => $this->clientSecret,
-                ],
-            ]
-        );
-
-        $this->tokenExpireTime->modify(
-            sprintf('+%s seconds', $tokenResponse['expires_in'])
-        );
-        $this->token = $tokenResponse['access_token'];
-        $this->tokenType = $tokenResponse['token_type'];
+        return $this->apiKey;
     }
```

Comment:
This whole method can be removed as the `apiKey` property can now be accessed directly

Reply:
@clone1018 nudging in case you missed this comment 🐠

Reply:
@clone1018 I need you to please make this change so that we can merge this in and release. Thank you.

Reply:
@clone1018 bumping this again

Reply:
Sorry man. I don't have the time or the project checked out. Feel free to make these changes yourself. Good luck!

Reply:
@darrynten if you want to add me as a contributor I can make these changes and push to the branch on the PR.